client/state/happychat/test/selectors.js

```diff
@@ -276,6 +277,17 @@ describe( 'selectors', () => {
 	} );
 
 	describe( '#getGroups()', () => {
+		let _window; // Keep a copy of the original window if any
```

Comment:
Thanks for the ping. That check was previously using an `abtest` check, which uses `window`. The abtest check was removed but it doesn't look like the window check was cleaned up. cc @rralian @jhnstn 

We should be able to safely remove it now in `isATEnabled`.  For other libs that use window I would recommend mocking out the entire lib instead of stubbing window. For example with analytics we can do:
```
useMockery( mockery => {
    mockery.registerMock( 'lib/analytics', {} );
 } );

```